### PR TITLE
ci(mypy): install dependencies to get access to all type definitions

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: mergify_engine/web/__init__.py [hunk 1]
```diff
@@ -46,20 +46,26 @@
 app.mount("/badges", badges.app)
 
 
+_AREDIS_STREAM: aredis.StrictRedis = None
+_AREDIS_CACHE: aredis.StrictRedis = None
```
Comment (sileht) [inline mergify_engine/web/__init__.py:50]: sniff...

File: tox.ini [hunk 2]
```diff
@@ -75,7 +75,6 @@ commands =
   black .
 
 [testenv:pep8]
-commands_pre =
```
Comment (sileht) [inline tox.ini:78]: ?

File: mergify_engine/web/__init__.py (hunk 1 above)
Comment (jd) [reply mergify_engine/web/__init__.py:50]: one could argue it's slightly cleaner that attaching things to an object like a cow-boy

File: tox.ini (hunk 2 above)
Comment (jd) [reply tox.ini:78]: Read the commit title; we need the deps to be installed to have the definition of the libraries we depends on. :)

File: mergify_engine/web/__init__.py (hunk 1 above)
Comment (sileht) [reply mergify_engine/web/__init__.py:49]: That's fastapi recommendation !

**Review by sileht — APPROVED** [review]

**Pull request merged**